index.html

```diff
@@ -102,7 +102,7 @@
 
 .pagedjs_page {
 		display: block;
-    width: calc(100 / 23 * 21vw)!important;
+    width: calc(100 / 23 * 21)!important;
```

Comment:
what's the difference?

Reply:
it was my mistake, vw is relative to 1% of the width of the viewport, that's why it was so wide before, now it should represent A4 page size more or less